subprojects/parseq/src/test-utils/java/com/linkedin/parseq/ParSeqUnitTestHelper.java

```diff
@@ -231,6 +228,15 @@ public ScheduledExecutorService getScheduler() {
     }
   }
 
+  //We don't want to use TestNG assertions to make the test utilities
+  // class useful for non TestNG users (for example, JUnit).
+  //Hence, we're writing our own private assertEquals method
+  static void assertEquals(Object o1, Object o2) {
```

Comment:
It's a package protected method, not exposed to clients. This class is not actively developed so it won't have new, sophisticated assertions in the future. Even if it has, we can always take in AssertJ as dependency. However today, given minimum amount of assertions, and that the class is not growing, it's best to avoid a dependency

Reply:
Got it. It is package protected, so we are good on this.